postgres/src/main/java/com/google/cloud/sql/postgres/SocketFactory.java

```diff
@@ -41,8 +42,10 @@
   private static final String POSTGRES_SUFFIX = "/.s.PGSQL.5432";
 
   private static final String INSTANCE_PROPERTY_KEY = "cloudSqlInstance";
+  private static final String IP_TYPES_KEY = "cloudsqlIpTypes";
```

Comment:
Are you open to renaming this to just `ipTypes`? If not, can we at least use `cloudSqlIpTypes` to be remain consistent?

Reply:
done, let's go with ipTypes.